src/main/java/com/currencycloud/client/CurrencyCloudClient.java

```diff
@@ -40,7 +40,7 @@
             "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}",
             Pattern.CASE_INSENSITIVE
     );
-    private static final String userAgent = "CurrencyCloudSDK/2.0 Java/0.7.8";
+    private static final String userAgent = "CurrencyCloudSDK/2.0 Java/0.9.1";
```

Comment:
We should define the version elsewhere and then link it to the UserAgent through interpolation.

Reply:
Agree. Ideally it should be derived from the pom. ToDO